composer.json

```diff
@@ -24,14 +24,14 @@
     },
     "require": {
         "php": "^8.0",
-        "illuminate/support": "^8.24|^9.0",
-        "imliam/laravel-blade-helper": "^1.4"
+        "illuminate/support": "^8.24|^9.0|^10.0",
+        "illuminate/view": "^8.24|^9.0|^10.0"
     },
     "require-dev": {
-        "orchestra/testbench": "^6.23|^7.0",
-        "nunomaduro/larastan": "^1.0",
-        "pestphp/pest": "^1.2",
-        "pestphp/pest-plugin-laravel": "^1.0",
+        "orchestra/testbench": "^6.23|^7.0|^8.0",
+        "nunomaduro/larastan": "^1.0|^2.4",
+        "pestphp/pest": "^1.2|^2.0",
+        "pestphp/pest-plugin-laravel": "^1.0|^2.0",
```

Comment:
I think for these dev dependencies we only need the latest version? Or do the 2.x versions of larastan & pest not support older versions of Laravel?

Reply:
Yes. Example: Larastan drop support Laravel 8.x, Pest Laravel drop 9.x and 8.x in version 2.0 😅

Reply:
I see, thanks!